src/OSPSuite.R/Services/CsvSeparatorSelector.cs

```diff
@@ -0,0 +1,12 @@
+﻿using OSPSuite.Infrastructure.Import.Services;
+
+namespace OSPSuite.R.Services
+{
+   public class CsvSeparatorSelector : ICsvSeparatorSelector
+   {
+      public char? GetCsvSeparator(string fileName)
+      {
+         return ',';
+      }
+   }
+}
```

Comment:
don't understand the purpose of this function. `fileName `is not used, it's declared as `char?` but always returns the same constant value - and why is it needed at all in R?

Reply:
> don't understand the purpose of this function. `fileName `is not used, it's declared as `char?` but always returns the same constant value - and why is it needed at all in R?

This is an interface that already existed. In PKSim, for example, it will show the first few lines to the user and ask, what separator would you like to use so you can select from a list. On R, there should be a way to do something similar (selecting the separator) but as explained before, I created an issue (#1166) for this, rather than tackling the problem now since I want to discuss with Pavel or other R users how this is more convenient for them. This is the reason why it is on a different issue. After implementing this issue, the function should now retrieve the same char always, it should be editable from settings or by using some setters but for now, I did not want to focus on this but rather I would like to keep it on a different issue. If you would prefer to have everything together, well, we also could. The param (fileName) is only there because of the existing interface, so from R no use, but from PKSim/MoBi it is indeed used.

Reply:
Why do we need this `CsvSeparatorSelector `in R at all?

Reply:
ICsvSeparatorSelector is a dependencie of CsvDataSourceFile (This is a class on the core). Normally, the ICsvSeparatorSelector is intended to handle user interaction (selection of the separator). The interface is on the core but the concrete implementation for PKSim or MoBi is on UI. This implementation is not available on R and either we decide to change this on the future or we have a different implementation for R. In any case, we need to be able to ask the user what separator does he want to use for a given csv file.

Reply:
>In any case, we need to be able to ask the user what separator does he want to use for a given csv file.

Why? In R I would expect that separaor is just set via scripting and not via user prompt.

Reply:
Exactly, we will have a wrapper on the R side. This wrapper should prepare the R implementation of ICsvSeparatorSelector to return whatever the script passed. This should be implemented on the other issue but for now, just to have something to start with, I am always using ','. Later, this implementation should be flexible. We could discuss it on a call if you prefer.

Reply:
Some changes that might explain a bit on #1170

Reply:
yes, with the last commit in #1170 it becomes much more clear. Thx!